### DISCUSSION TITLE
Why is no-mixed-operators deprecated / stylistic?

### DISCUSSION THREAD

**Discussion by MangoMarcus** [eslint/eslint#18408] [Q&A] [answered]
Hi all, I noticed the [no-mixed-operators](https://eslint.org/docs/latest/rules/no-mixed-operators) rule has been deprecated in favour of the [stylistic equivolent](https://eslint.style/rules/js/no-mixed-operators), I'm just wondering what the rationale is for considering it "stylistic" ?

Operator precedence is a common cause of bugs and it's an easy mistake to make. Adding parenthesis goes beyond a styling decision, it's used to make the developer's intention clear at a glance without having to worry about memorising the precedence of each operator in Javascript.

Thanks!

**Comment by mdjermanovic** [comment, marked as answer]
Indeed, we didn't consider this rule as just stylistic, Please see the discussion in https://github.com/eslint/eslint/issues/17522 for the rationale for deprecating this rule in the core and moving it to the stylistic plugin (in particular, https://github.com/eslint/eslint/issues/17522#issuecomment-1764746404 and https://github.com/eslint/eslint/issues/17522#issuecomment-1764348498).

**Reply by MangoMarcus** [reply]
Thanks for the speedy response!
